book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rental << rental
+  end
```

Comment:
- [ ] This method should serve as the method that allows us to create a rental given book, date, and person. Since we already have access to the book (through `self`) we just need to pass in person and date as parameters and create a rental object. Kindly refactor this method.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```